### DISCUSSION TITLE
Unable to resolve action `fscarmen/warp-on-actions@v2`, unable to find version `v2`

### DISCUSSION THREAD

**Discussion by ghost** [upptime/upptime#919] [Q&A] [answered]
Hello, I recently forked your template, made some edits, and the update of the .upptimerc.yml file failed. I got this Error: Unable to resolve action `fscarmen/warp-on-actions@v2`, unable to find version `v2`

What happaned?

Log:

##[debug]Starting: Set up job
Current runner version: '[2](https://github.com/Iliya5digital/sites/actions/runs/7602853489/job/20703886829#step:1:2).[3](https://github.com/Iliya5digital/sites/actions/runs/7602853489/job/20703886829#step:1:3)11.0'
Operating System
Runner Image
Runner Image Provisioner
GITHUB_TOKEN Permissions
Secret source: Actions
##[debug]Primary repository: Iliya5digital/sites
Prepare workflow directory
##[debug]Creating pipeline directory: '/home/runner/work/sites'
##[debug]Creating workspace directory: '/home/runner/work/sites/sites'
##[debug]Update context data
##[debug]Evaluating job-level environment variables
##[debug]Evaluating job container
##[debug]Evaluating job service containers
##[debug]Evaluating job defaults
Prepare all required actions
Getting action download info
Error: Unable to resolve action `fscarmen/warp-on-actions@v2`, unable to find version `v2`
##[debug]GitHub.DistributedTask.WebApi.UnresolvableActionDownloadInfoException: Unable to resolve action `fscarmen/warp-on-actions@v2`, unable to find version `v2`
##[debug]   at GitHub.Services.WebApi.VssHttpClientBase.HandleResponseAsync(HttpResponseMessage response, CancellationToken cancellationToken)
##[debug]   at GitHub.Services.WebApi.VssHttpClientBase.SendAsync(HttpRequestMessage message, HttpCompletionOption completionOption, Object userState, CancellationToken cancellationToken)
##[debug]   at GitHub.Services.WebApi.VssHttpClientBase.SendAsync[T](HttpRequestMessage message, Object userState, CancellationToken cancellationToken)
##[debug]   at GitHub.Services.WebApi.VssHttpClientBase.SendAsync[T](HttpMethod method, IEnumerable`1 additionalHeaders, Guid locationId, Object routeValues, ApiResourceVersion version, HttpContent content, IEnumerable`1 queryParameters, Object userState, CancellationToken cancellationToken)
##[debug]   at GitHub.Runner.Worker.ActionManager.GetDownloadInfoAsync(IExecutionContext executionContext, List`1 actions)
##[debug]   at GitHub.Runner.Worker.ActionManager.GetDownloadInfoAsync(IExecutionContext executionContext, List`1 actions)
##[debug]   at GitHub.Runner.Worker.ActionManager.PrepareActionsRecursiveAsync(IExecutionContext executionContext, PrepareActionsState state, IEnumerable`1 actions, Int[32](https://github.com/Iliya5digital/sites/actions/runs/7602853489/job/20703886829#step:1:36) depth, Guid parentStepId)
##[debug]   at GitHub.Runner.Worker.ActionManager.PrepareActionsAsync(IExecutionContext executionContext, IEnumerable`1 steps, Guid rootStepId)
##[debug]   at GitHub.Runner.Worker.JobExtension.InitializeJob(IExecutionContext jobContext, AgentJobRequestMessage message)
##[debug]Finishing: Set up job

**Comment by AnandChowdhary** [comment, marked as answer]
Closing as duplicate of https://github.com/upptime/upptime/issues/918, let's discuss there! :)
